simple-umbraco-webapp/README.md

```diff
@@ -0,0 +1,10 @@
+# Build a simple Umbraco Web App 
```

Comment:
@SunBuild The other template is called "umbraco-cms-webapp-redis-cache" and this one "simple-umbraco-webapp" so they won't show next to each other in the root list of the repo. I suggest calling both something like 

"umbraco-webapp-cluster" and "umbraco-webapp-simple" so they appear together in the list. See example templates following this approach below
![image](https://cloud.githubusercontent.com/assets/6164178/16158082/71c9e2dc-3470-11e6-8fca-7dc4e915bee7.png)